admin_guide/topics/crd_overview.adoc

```diff
@@ -0,0 +1,42 @@
+////
+custom resource definition overview
+
+Module included in the following assemblies:
+
+* admin_guide/custom_resource_definitions.adoc
+* dev_guide/creating_crd_objects.adoc
+////
+
+In the Kubernetes API a resource is an endpoint that stores a collection of API objects of a certain kind.
+For example, the built-in pods resource contains a collection of Pod objects.
+
+A _custom resource_ is an object that extends the Kubernetes API or allows you to
+introduce your own API into a project or a cluster.
+
+ifeval::["{context}" == "admin-guide-custom-resources"]
+A _custom resource definition_ (CRD) file defines your own object kinds and lets
+the API Server handle the entire lifecycle. Deploying a CRD into the cluster 
+causes the Kubernetes API server to begin serving the specified custom resource.
+
+When you create a new custom resource definition (CRD), the Kubernetes API Server
+reacts by creating a new RESTful resource path, that can be accessed by an entire cluster
+or a single project (namespace). As with existing built-in objects, deleting a
+project deletes all custom objects in that project.
+
+If you want to grant access to the CRD to users, use cluster role aggregation to
+grant access to users with the admin, edit, or view default cluster roles. 
+Cluster role aggregation allows the insertion of custom policy rules into these
+cluster roles. This behavior seamlessly integrates the new resource into the 
+cluster's RBAC policy as if it was a built-in resource.
+endif::[] 
+
+ifeval::["{context}" == "dev-guide-crd"]
+A _custom resource definition_ (CRD) file defines your own object kinds and lets
+the API Server handle the entire lifecycle.
+endif::[] 
+
+[NOTE]
+====
+While only cluster admins can create CRDs, any user with the permission to do
+so can create an object from a CRD.
```

Comment:
can create an instance of a CRD

Reply:
@enj, so is it ever right to say that you can create an object from a CRD? That language shows up in a couple of other spots.

Reply:
Based on https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#create-custom-objects I think it is fine as-is.